plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayer.java

```diff
@@ -178,11 +174,14 @@ private void addNavigationLayer(Drawable navigationDrawable) {
   }
 
   private void addAccuracyLayer(float accuracyAlpha, @ColorInt int accuracyColor) {
-    FillLayer locationAccuracyLayer = new FillLayer(
-      LocationLayerConstants.LOCATION_ACCURACY_LAYER, LocationLayerConstants.LOCATION_ACCURACY_SOURCE
+    CircleLayer locationAccuracyLayer = new CircleLayer(
+      LocationLayerConstants.LOCATION_ACCURACY_LAYER, LocationLayerConstants.LOCATION_SOURCE
     ).withProperties(
-      PropertyFactory.fillColor(accuracyColor),
-      PropertyFactory.fillOpacity(accuracyAlpha)
+      PropertyFactory.circleColor(accuracyColor),
```

Comment:
instead of the whole `PropertyFactory.circleColor(accuracyColor)` you can statically import `circleColor(accurancyColor)` instead. Reads a bit more easy.